packages/taquito/src/contract/rpc-contract-provider.ts

```diff
@@ -395,6 +400,127 @@ export class RpcContractProvider extends Provider implements ContractProvider, S
     return new TransactionOperation(hash, content, source, forgedBytes, opResponse, context);
   }
 
+  /**
+   *
+   * @description Stake tz from current address to a specific address. Built on top of the existing transaction operation
+   *
+   * @returns An operation handle with the result from the rpc node
+   *
+   * @param Stake pseudo-operation parameter
+   */
+  async stake(params: StakeParams) {
+    if (params.to === undefined) {
```

Comment:
might be more robust to evaluate `if(params.to?)` to exclude other possibilities like `null`  (same apply to other two pseudo ops)